src/resources_components.rs

```diff
@@ -5,22 +5,6 @@ use std::{
 
 use bevy::prelude::*;
 
-/// Internal component which tracks the last position at which the entity was updated in the tree.
-#[derive(Component)]
-pub struct MovementTracked<T> {
```

Comment:
This appears to no longer be used after https://github.com/laundmo/bevy-spatial/pull/14, so I deleted it.